Keas.Mvc/ClientApp/util/permissions.ts

```diff
@@ -20,6 +20,11 @@
         return r.some(a => permissionArray.includes(a));
     }
 
+    public static canEditPeople(r: string[]) {
+        const permissionArray = ['DepartmentalAdmin'];
```

Comment:
@laholstege Can you use the Role.Codes? Or does that cause problems?
Role.Codes.DepartmentalAdmin